### DISCUSSION TITLE
[Feature Request] Show Discard after taking a meld.

### DISCUSSION THREAD

**Discussion by SyxP** [Equim-chan/mjai-reviewer#109] [Ideas]
Sometimes the recommended move is to Chii/Pon/Kan. Is it possible to subsequently display the tile it wishes to discard?

**Comment by Equim-chan** [comment]
It is possible, but it requires a lot of changes, and the details table won't work. Since Mortal only outputs a single action, if you want to know its next move, you'll have to fork the state and commit that premise action (chi/pon/riichi).
